eks_addon/common.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Common functions for EKS Add-on preparation and testing
+
+# Enable bash strict mode
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CHART_DIR="$SCRIPT_DIR/../helm-charts/splunk-otel-collector"
+ORIG_CHART_DIR="$SCRIPT_DIR/../helm-charts/splunk-otel-collector"
+BUILD_DIR="${SCRIPT_DIR}/build"
```

Comment:
Is there any potential to conflict in the future where `./build` would be used for something else? I'm wondering if it'd be worthwhile to name it more specifically since we're wiping it out every time the `prepare_chart` function is run.